matrix/matrix.py

```diff
@@ -62,20 +76,15 @@ def _add_matrix(self, other):
             Matrix: the result of the addition.
         """
         dimension = len(self.tuples)
+        empty_data = tuple([] for i in range(dimension))
```

Comment:
Actually all I meant here, that instead of
[0 i range(x)]
you can write
[0] * x

Reply:
what I need is a tuple of lists. Using the multiplication option causes reference errors (I tried it). Is `tuple([] for i in range(dimension))` one that bad?

Reply:
its not that bad, its just I saw you did a lot of changes because of my comment, so I wanted to explain what I meant you to do (very small change)